config/routes.rb

```diff
@@ -3,6 +3,7 @@
   resources :likes
   resources :posts
   resources :users
+
```

Comment:
Kindly ensure that in this file you only have routes that are necessary for this milestone only. You are suppose to create routes (show and index) for the users and posts controllers only. Kindly update to match the project requirements. :wink:

Reply:
Aye Aye Captain